change/@ni-nimble-components-5084859c-b0ef-4431-b684-b4733c6d7714.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm still seeing the buggy behavior when autocomplete is set to `none` in the current Storybook build.
https://60e89457a987cf003efc0a5b-nndckhaqkg.chromatic.com/?path=/docs/components-combobox--docs

https://github.com/user-attachments/assets/fe92ff9c-3bfd-45ce-bcc4-1cbab8d83dae

Reply:
So many code paths.... Fixed now. Had to make sure that when `options` is an unfiltered list, and we're searching for the index of some value, we treat disabled options as not found.